app/routes/wow.ilvl-export-search.tsx

```
@@ -312,15 +322,25 @@ const IlvlExportSearchComponent = () => {
             { label: 'MEDIUM', value: '1' },
             { label: 'LOW', value: '0' }
           ]}
-          onChange={(e) => setFormValues(prev => ({ ...prev, populationBlizz: parseInt(e.target.value) }))}
+          onChange={(e) =>
+            setFormValues((prev) => ({
+              ...prev,
+              populationBlizz: parseInt(e.target.value)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)